docs/dg/dev/internationalization-and-multi-store/add-new-store-in-multi-db-setup.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Adding new store in multi-db setup"
+description: Adding and deploying a new store in multi-db store setup requires additional steps and preparation. This guideline contains all the best practices you need to know.
+last_updated: Dec 02, 2024
+template: howto-guide-template
+originalLink: 
+originalArticleId: 
+redirect_from:
+---
+
+Setting up a new store in an existing multi-DB environment requires a well-prepared plan, to avoid disrupting the data and operations of existing stores. This guide provides a detailed procedure to release a new store within a region that already hosts other stores, ensuring a non-destructive deployment.
```

Comment:
`non-destructive deployment` sounds like something related to `destructive deployment`. If it's not related, rephrase it

Reply:
Please check, I changed the sentence